### DISCUSSION TITLE
Add jupyter-notebook training + test example

### DISCUSSION THREAD

**Discussion by alexeyhorkin** [rwightman/efficientdet-pytorch#245] [Ideas]
Wouldn't it be nice if I added a jupyter-notebook example how to train end test effdet models on a custom dataset?
Is it would be useful?

**Comment by alexeyhorkin** [comment]
@rwightman =)

**Reply by rwightman** [reply]
@alexeyhorkin there's this example https://gist.github.com/Chris-hughes10/73628b1d8d6fc7d359b3dcbbbb8869d7  ... my training scripts aren't very notebook friendly right now and would require modifications, but this lightning based example should work

**Comment by rayennecandido** [comment]
amazing repo
